internal/handlers/common/serverstatus.go

```diff
@@ -19,14 +19,15 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/FerretDB/FerretDB/internal/clientconn"
 	"github.com/FerretDB/FerretDB/internal/types"
 	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/util/version"
 )
 
 // ServerStatus returns a common part of serverStatus command response.
-func ServerStatus(startTime time.Time) (*types.Document, error) {
+func ServerStatus(startTime time.Time, cm *clientconn.ConnMetrics) (*types.Document, error) {
```

Comment:
Here (`common` package) we need to import the `clientconn.ConnMetrics`, because it's required to provide the metrics to the `serverStatus` cmd

Reply:
I don't think we should import `common` from `clientconn`, yeah.

Let's try passing commands as `[]string`